functions.py

```diff
@@ -0,0 +1,162 @@
+import csv
+import datetime
+import exceptions_tests
+import os #this is needed for using directory paths and manipulating them
+import sys
+from pip._vendor.distlib.compat import raw_input
+
+class Employee(object):
+
+    def __init__(self, employee_id, name, phone, age):
+        self.employee_id = employee_id
+        self.name = name
```

Comment:
Is it full name? Or first name?

Reply:
It's the full name of the employee - I need to add it to the name function in exceptions test file :)

Reply:
change from 'name' to 'full_name'

Reply:
done :)